config.sh.example

```diff
@@ -39,3 +39,20 @@ export API_NODEPORT=31123
 
 # the external port on which to connect to the managed openvpn server
 export OPENVPN_NODEPORT=31124
+
+# cluster id
+export CLUSTER_ID=cluster_id
+
+# Used for etcd config
```

Comment:
How is this optionally enabled?  It looks like the support requires backup/restore to be enabled.

Reply:
backup and restore creds are optional (but you still need to provide garbage values). This needs to be required but we want to provide a way for people to still have this working while they work on getting cos buckets or change process. Images are not.